Package.swift

```diff
@@ -19,11 +19,11 @@ let package = Package(
     dependencies: [
         .package(
             url: "https://github.com/apple/swift-argument-parser",
-            .upToNextMinor(from: "0.3.0")
+            .upToNextMinor(from: "1.0.2")
         ),
         .package(
-            url: "https://github.com/tuist/XcodeProj.git",
-            .upToNextMinor(from: "7.18.0")
+            url: "https://github.com/marinofelipe/XcodeProj.git",
```

Comment:
This currently points to a fork, since PathKit, a dependency of `XcodeProj` wasn't being correctly built on Xcode 13. It should be updated to a mainstream XcodeProj release when feasible.